sql/src/main/java/org/apache/druid/sql/calcite/schema/SegmentMetadataCache.java

```diff
@@ -707,14 +709,19 @@ private long recomputeIsRealtime(ImmutableSet<DruidServerMetadata> servers)
   private Set<SegmentId> refreshSegmentsForDataSource(final String dataSource, final Set<SegmentId> segments)
       throws IOException
   {
+    final long startTimeNanos = System.nanoTime();
```

Comment:
Since you've given this enough thought - probably Guava's `StopWatch` is a good utility to use here. It is also used at a couple of more place and provides the elapsed and conversion methods as well

Reply:
You can consider this change, incase you're going to make other changes to the PR too. no point in running CI for this :)

Reply:
How is this better compared to using nano time? api wise or better performant?